beamer/tests/agent/unit/test_invalidate.py

```diff
@@ -73,3 +76,72 @@ def test_handle_hash_invalidated():
 
     # Check that a non-matching event does not invalidate the claim
     assert claim_2.is_claimer_winning  # pylint:disable=no-member
+
+
+def test_maybe_invalidate_claim_wrong_fill_id():
+    context, config = make_context()
+
+    request = make_request(valid_until=int(time.time() * 2))
```

Comment:
Is it necessary to make the request valid here? Would it change anything if we leave it with the default?

Reply:
The default is currently some fixed number that's in the past, so we need this here.